xblock/test/test_parsing.py

```diff
@@ -344,7 +344,7 @@ def test_dict_and_list_export_format(self):
     def test_unknown_field_as_attribute_raises_warning(self, parameter_name):
         with mock.patch('logging.warning') as patched_warn:
             block = self.parse_xml_to_block(f"<leaf {parameter_name}='something irrelevant'></leaf>")
-            patched_warn.assert_called_once_with("XBlock %s does not contain field %s", type(block), parameter_name)
+            patched_warn.assert_called_once_with("%s does not contain field %s", type(block), parameter_name)
```

Comment:
This warning was incorrect when the object was an XBlockAside.

I considered changing it to "Blocklike", but then the message would be:

`Blocklike FooBlock does not contain field bar`

which isn't any better than:

`FooBlock does not contain field bar`.

I could be convinced otherwise, esp if folks feel that the former is more regex-able.